libraries/grpc-sdk/src/interfaces/Indexable.ts

```diff
@@ -0,0 +1,7 @@
+export type Indexable = { [key: string]: any };
+export type Params = Indexable;
+export type Document = Indexable;
+export type Route = Indexable;
+export type Headers = Indexable;
+export type Context = Indexable;
+export type Query = Indexable;
```

Comment:
Export * from `src/interfaces/index.ts` (and import Query directly from index.ts in ConduitActiveSchema)
Add new line at EOF.